### PR TITLE
Improve the PB2002 data section

### PR DESCRIPTION
**Pull request by core-man**

### CODE REVIEW

**Comment by seisman** [comment]
The two download links don't look good.

image

File: source/dataset/PB2002/index.rst [hunk 1]
```diff
@@ -10,19 +10,16 @@ PB2002 是常用的一个全球板块边界数据，其包含了14个大板块
 数据下载
 --------
 
-原始数据的格式无法直接用于GMT绘图。这里我们从官方网站下载了 ``PB2002_plates.dig.txt``
-和 ``PB2002_boundaries.dig.txt`` 数据，并将其段头记录做了修改以供GMT用户使用。
+原始数据的格式无法直接用于GMT绘图。这里我们从官方网站下载了 :file:`PB2002_plates.dig.txt`
+和 :file:`PB2002_boundaries.dig.txt` 数据，并将其段头记录做了修改以供GMT用户使用。
+可供GMT直接使用的数据下载地址: :download:`PB2002_plates.dig.txt` | :download:`PB2002_boundaries.dig.txt`
```
Comment (core-man) [inline source/dataset/PB2002/index.rst:15]:
```suggestion

可供GMT直接使用的数据下载地址: :download:`PB2002_plates.dig.txt` | :download:`PB2002_boundaries.dig.txt`
```

File: source/dataset/PB2002/index.rst [hunk 2]
```diff
@@ -10,19 +10,17 @@ PB2002 是常用的一个全球板块边界数据，其包含了14个大板块
...
 
-两个数据是等效的，区别在于 **PB2002_plates.dig.txt** 中数据分为52段，
-每段数据对应一个板块边界闭合多边形，每段数据的段头记录为该段数据的板块名称缩写。
-
-**PB2002_boundaries.dig.txt** 中数据分为229段，其将相邻板块的共同边界分为不同的段列出来。
-段头记录中包含了两个板块的名称缩写，以及板块数据来源的参考文献。
+- :file:`PB2002_plates.dig.txt` 中数据分为 52 段，每段数据对应一个板块边界闭合多边形，
+  每段数据的段头记录为该段数据的板块名称缩写。
+- :file:`PB2002_boundaries.dig.txt` 中数据分为 229 段，其将相邻板块的共同边界分为不同的段列出来。
+  段头记录中包含了两个板块的名称缩写，以及板块数据来源的参考文献。
```
Comment (core-man) [inline source/dataset/PB2002/index.rst:23]:
```suggestion
- :file:`PB2002_plates.dig.txt`\ ：数据分为 52 段，每段数据对应一个板块边界闭合多边形，
  每段数据的段头记录为该段数据的板块名称缩写。
- :file:`PB2002_boundaries.dig.txt`\ ：数据分为 229 段，其将相邻板块的共同边界分为不同的段列出来。
  段头记录中包含了两个板块的名称缩写，以及板块数据来源的参考文献。
```

File: source/dataset/PB2002/index.rst [hunk 3]
```diff
@@ -10,19 +10,17 @@ PB2002 是常用的一个全球板块边界数据，其包含了14个大板块
 数据下载
 --------
 
-原始数据的格式无法直接用于GMT绘图。这里我们从官方网站下载了 ``PB2002_plates.dig.txt``
-和 ``PB2002_boundaries.dig.txt`` 数据，并将其段头记录做了修改以供GMT用户使用。
+原始数据的格式无法直接用于GMT绘图。这里我们从官方网站下载了 :file:`PB2002_plates.dig.txt`
+和 :file:`PB2002_boundaries.dig.txt` 数据，并将其段头记录做了修改以供GMT用户使用。
 
-可供GMT直接使用的数据下载地址:
+可供GMT直接使用的数据下载地址: :download:`PB2002_plates.dig.txt` | :download:`PB2002_boundaries.dig.txt`
```
Comment (core-man) [inline source/dataset/PB2002/index.rst:16]:
```suggestion
可供GMT直接使用的数据下载地址：\ :download:`PB2002_plates.dig.txt` | :download:`PB2002_boundaries.dig.txt`
```

**Review by seisman — APPROVED** [review]

**Pull request merged**